### ISSUE TITLE
reduce the size of photosphere in the RPacketPlotter Plot

### ISSUE THREAD

**Issue by jayantbhakar** [tardis-sn/tardis#2116]
RPacketPlotter in PR #2119 creates a plot where every element on the plot is equally scaled. This makes the photosphere cover most of the space in the plot. We want to reduce the area covered by the photosphere in the plot.

![Image](https://user-images.githubusercontent.com/60151232/185781401-7c2f7c9d-13e2-4213-b808-8348eebe36b3.png)

**Comment by notinrange** [comment]
@jayantbhakar In which amount of small scale do you want to reduce photosphere size visualization? 
In this visualization given to you, you want to reduce the size of a grey area right?

**Comment by jayantbhakar** [comment]
Yes, the grey area, starting from 0 to 10000 km/s. One idea is to make the plot start from 5000 km/s till the end, this would remove a circular area from the photosphere of radius 0 to 5000 km/s.

**Comment by ishika-2110** [comment]
Hey can i work on this issue, if it's already not been assigned to someone else?

**Comment by MariamH78** [comment]
Hey! I'd like to work on this issue but I'm having a little trouble understanding the objectives and the workflow on Github is still not quite familiar to me so I have a couple of questions:
    1. Does the size of the photosphere have any physical significance? Or will any moderately-sized circle do? And if its dimensions _are_ important, how were you planning to go about this?
    2. Should I fork your branch from that PR, work on it, then request merging there as well?

**Comment by jayantbhakar** [comment]
Hi @MariamH78. Glad to see your interest in this issue.
To your first point, we wanted to scale down the axis starting from the origin and until the photosphere boundary. This downscaling should be seen in the axis labels as well so that the user can get an exact idea about the plot. The main idea behind the issue is that the size of the photosphere needs to be reduced as it is taking up a lot of space right now.
To your second point, yes you can fork from my branch in the PR.

**Comment by MariamH78** [comment]
Done! I created a [pull request](https://github.com/jayantbhakar/tardis/pull/1#issue-1638334033) on your PR. Thanks for the help!

**Comment by mohakmalviya** [comment]
Hello sir! i would like to work on this issue if it's not been resolved yet!

**Comment by rudrakshnalbalwar** [comment]
Hey, Can I work on this issue ?

**Comment by Manivenkat3612** [comment]
Hi @jayantbhakar. I see this issue is still open and I'd like to try implementing the scaling for the photosphere as suggested. Is it okay for me to work on this?

**Comment by andrewfullard** [comment]
Please give it a try.
